contracts/bond/Bond.sol

```diff
@@ -188,12 +202,13 @@ contract Bond is
     }
 
     /**
-     *  @notice Moves any remaining collateral to the Treasury and pauses the bond.
+     *  @notice Moves all remaining collateral to the Treasury and pauses the bond.
+     *
      *  @dev A fail safe, callable by anyone after the Bond has expired.
-     *       If Ownership is lost, this can be used to move all remaining collateral to the Treasury,
+     *       If control is lost, this can be used to move all remaining collateral to the Treasury,
      *       after which petitions for redemption can be made.
-     *  @dev Expiry operates separately to pause, so a paused contract can be expired (fail safe for loss of
-     *       Ownership).
+     *
+     *  Expiry operates separately to pause, so a paused contract can be expired (fail safe for loss of control).
```

Comment:
If @dev is detailed description about function process/execution/conditions, why "Expiry operates..." is not part of above lines "A fail safe... If control..." and its indentation is different from it?

Reply:
Interesting, after looking through the OZ example, I had the impression that all text following a tag (e.g. `@dev`) and before the next tag is covered by that tag.

This would allow for multiple paragraphs to be covered by a single tag, rather then starting each paragraph with the tag.

Example being the contract level comment and `onlyRole` from AccessControl.sol
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/AccessControl.sol

As I've not yet used the `solc` generation tool, I can't confirm whether my assumption is good or not 🤔